CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Error/ErrorScreenViewController.swift

```diff
@@ -35,6 +30,17 @@ class ErrorScreenViewController: UIViewController {
             secondTitle: NSLocalizedStringPreferredFormat(
                 "ginicapture.error.backToCamera",
                 comment: "Back to camera"))
+        view.translatesAutoresizingMaskIntoConstraints = false
+        if viewModel.isEnterManuallyHidden() == false {
```

Comment:
`if viewModel.isEnterManuallyHidden() == false {
             view.enterButton.isHidden = false
         } else {
             view.enterButton.isHidden = true
         }`
         
         This whole block could be written like this:
         
         `view.enterButton.isHidden = viewModel.isEnterManuallyHidden()`
         
         Same for the one below